src/styles/utils/veriables.scss

```diff
@@ -0,0 +1,7 @@
+$card-width: 200px;
+$text-color-1: #060b35;
+$text-color-2: #616070;
+$button-color: #00acdc;
+$button-text-color: #fff;
+$tramsform-time: 300ms;
```

Comment:
There is a typo in the variable name `$tramsform-time`. It should likely be `$transform-time`. Ensure this variable is correctly referenced in other SCSS files.